cpp/models/abm/location.h

```diff
@@ -255,6 +271,7 @@ class Location
     LocationCapacity m_capacity;
     bool m_capacity_adapted_transmission_risk;
     std::array<int, size_t(InfectionState::Count)> m_subpopulations;
+    TimeSeries<double> m_subpopulations_time_series;
```

Comment:
Couldn't we reduce this to one member and access the current state from the last one?

Reply:
The 'm_subpopulations' is now removed. All relevant functions are updated accordingly.